### ISSUE TITLE
move some functions from DatabaseConnector to SqlRender

### ISSUE THREAD

**Issue by vojtechhuser** [OHDSI/SqlRender#199]
In my view, it would be more logical if functions listed below from DatabaseConnector 

```
 renderTranslateExecuteSql() #Render, translate, execute SQL code
 renderTranslateQuerySql() #Render, translate, and query to data.frame

```

live in SqlRender

next to function like loadRenderTranslateSql

**Comment by schuemie** [comment]
Well, they're function that logically could be in either package. If we move them to SqlRender, SqlRender would become dependent on DatabaseConnector. DatabaseConnector is already dependent on SqlRender, so this would create a circular dependency, 

What would be the benefit of having them in SqlRender?

**Comment by vojtechhuser** [comment]
I don't see circular dependency as a problem. They are intertwined anyway. 

For a new developer, having all of them in one place is an advantage. One other option is to at least refer to them (in both directions) in help text of each function. I did not know about them until I saw your recent code in Eunomia. I wish I knew about them much earlier. (well, or just when they were added to DatabaseConnector).
